src/com/jetbrains/GraphicsUtils.java

```diff
@@ -74,7 +74,8 @@ public interface ConstrainableGraphics2D {
     }
 
     /**
-     * Checks whether display is built-in.
+     * Checks whether the display is built-in.
+     * Supported only on macOS (can be implemented in other OS if necessary).
```

Comment:
> can be implemented in other OS if necessary

I'm not sure if this can be reliably implemented on Linux. Let's not give promises here :-)